Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -51,6 +51,7 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
     @IBOutlet var titleField: UITextField!
     @IBOutlet var bodyField: UITextView!
 
+    private var template: String = ""
```

Comment:
Nit: you can omit the `: String`; it is inferred.

Reply:
Would we not just rather this be nil if there is no template? Feels like a tidier solution in my opinion

Reply:
@Sherlouk Updated!

Reply:
@Huddie Swish! If you give us a shout once you're looking for another review and I can do a full sweep at that point, hard to keep track of all these individual discussions at this point 😅